### ISSUE TITLE
Plotting multipolygon objects

### ISSUE THREAD

**Issue by mkborregaard** [JuliaGeo/GeoInterface.jl#42]
There seems to be an issue with plotting polygons that themselves consist of several polygons. Here is a dataset with multipolygons
```julia
julia> regions = Shapefile.shapes(Shapefile.Table("regions1984.shp"));

julia> julia> regions[1]
Polygon(15493 Points)

julia> plot(regions, fill_z = 1:length(regions))
Error showing value of type Plots.Plot{Plots.GRBackend}:
ERROR: BoundsError: attempt to access 20-element UnitRange{Int64} at index [55]
```
Here's the data, if that's useful https://www.dropbox.com/s/xbvmb496ml7hr7y/Archive.zip?dl=0

Should this issue be here or on Shapefile?

**Comment by visr** [comment]
It's probably an issue here, though I don't really know what's wrong.

Your example works without `fill_z = 1:length(regions)`. The problem seems to be that there are `length(regions)` number of multipolygons, but many more polygons. At least the outer rings of each polygon in a multipolygon should be combined, with a NaN inserted as a separator:

https://github.com/JuliaGeo/GeoInterface.jl/blob/2d60c435755d267570097aee941048386568c609/src/plotrecipes.jl#L34-L43

**Comment by mkborregaard** [comment]
Yes, we may have to handle the repeats of polygon attributes ourselves internally - like I did for the multi lines that constitute branches on a phylogenetic tree https://github.com/EcoJulia/Phylo.jl/blob/dev/src/plot.jl#L180-L189

**Comment by evetion** [comment]
Will be solved by #45.
